lightning/src/routing/network_graph.rs

```diff
@@ -1414,31 +1414,31 @@ impl NetworkGraph {
 							},
 							last_update_message
 						};
-						$target = Some(updated_channel_update_info);
+						Some(updated_channel_update_info);
```

Comment:
Will need to drop the semi-colon. Though could make this macros a single expression, FWIW.